migration/sql-files/035-unique_constraint_default_role_mapping.sql

```diff
@@ -0,0 +1,2 @@
+-- We must not allow multiple creations of role mapping for a given type of resource with the same 'from' and 'to' roles.
+alter table default_role_mapping add constraint unique_default_role_mapping UNIQUE (resource_type_id, from_role_id, to_role_id);
```

Comment:
Thankfully, we don't have an exposed API for this, else you would have had to consider migration.

Reply:
right, that's something we would have needed to handle, too. Same would need to apply for "custom" role mappings, but I have not started work on those ones yet. Probably in another PR